test/performance/latency_test.go

```diff
@@ -32,7 +32,7 @@ import (
 )
 
 func TestTimeToServeLatency(t *testing.T) {
-	perfClients, err := Setup(t.Logf, true)
+	perfClients, err := Setup(t.Logf, t.Name())
```

Comment:
Do we want to disable Prometheus for `TestTimeToServeLatency`?

Reply:
Its not being used in the test. We can re-add it later if we want.